### PR TITLE
Engdocs 1031

### PR DESCRIPTION
**Pull request by jerae-duffin**
Updated note for Mappings.

### CODE REVIEW

File: _data/toc.yaml [hunk 1]
```diff
@@ -1476,8 +1476,14 @@ manuals:
...
-  - path: /docker-hub/repos/
-    title: Repositories
+  - sectiontitle: Repositories
+    section:
+      - path: /docker-hub/repos/
+        title: Manage
+      - path: /docker-hub/repos/access/
+        title: Access
+      - path: /docker-hub/repos/configure/
+        title: Configuring
```
Comment (jerae-duffin) [inline _data/toc.yaml:1486]:
```suggestion
        title: Configure
```

File: docker-hub/members.md [hunk 2]
```diff
@@ -115,6 +115,8 @@ To remove a member from a specific team:
 2. Click on the **Teams** tab and select the team from the list.
 3. Click the **X** next to the user’s name to remove them from the team.
 4. When prompted, click **Remove** to confirm.
+<<<<<<< HEAD
```
Comment (craig-osterhout) [inline docker-hub/members.md:118]: Remove some fragments from a merge.

File: docker-hub/members.md [hunk 3]
```diff
@@ -115,6 +115,8 @@ To remove a member from a specific team:
...
+=======
```
Comment (craig-osterhout) [inline docker-hub/members.md:119]: Remove some fragments from a merge.

File: docker-hub/members.md [hunk 4]
```diff
@@ -132,4 +134,5 @@ The CSV file contains the following fields:
 To export a CSV file of the organization's members:
 
 1. Navigate to **Organizations** in [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"}, and select your organization.
-2. In the **Members** tab, select **Export members** to download the CSV file.
\ No newline at end of file
+2. In the **Members** tab, select **Export members** to download the CSV file.
+>>>>>>> 5c68a7ad02a3a842aca984dcb72d13732506c06f
```
Comment (craig-osterhout) [inline docker-hub/members.md:138]: Remove some fragments from a merge.

File: docker-hub/repos/access/index.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
+---
+description: Access repositories on Docker Hub
+keywords: Docker, docker, trusted, registry, accounts, plans, Dockerfile, Docker Hub, webhooks, docs, documentation, collaborators, viewing, searching, starring
+title: Access
+redirect_from:
```
Comment (craig-osterhout) [inline docker-hub/repos/access/index.md:5]: Remove `redirect_from:` if no redirects.

File: docker-hub/repos/configure/index.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
+---
+description: Configuring repositories on Docker Hub
+keywords: Docker, docker, trusted, registry, accounts, plans, Dockerfile, Docker Hub, webhooks, docs, documentation, creating, deleting, consolidating
+title: Configure
+redirect_from:
```
Comment (craig-osterhout) [inline docker-hub/repos/configure/index.md:5]: Remove `redirect_from:` if no redirects.

File: docker-hub/repos/index.md [hunk 7]
```diff
@@ -0,0 +1,108 @@
+---
+description: Using repositories on Docker Hub
+keywords: Docker, docker, trusted, registry, accounts, plans, Dockerfile, Docker Hub, webhooks, docs, documentation, manage, repos
+title: Repositories
```
Comment (craig-osterhout) [inline docker-hub/repos/index.md:4]: The `Repositories` title is inconsistent with the `Manage` title in the TOC.

File: docker-hub/repos/configure/index.md [hunk 8]
```diff
@@ -0,0 +1,59 @@
+---
+description: Configuring repositories on Docker Hub
+keywords: Docker, docker, trusted, registry, accounts, plans, Dockerfile, Docker Hub, webhooks, docs, documentation, creating, deleting, consolidating
+title: Configure
```
Comment (craig-osterhout) [inline docker-hub/repos/configure/index.md:4]: Similar to another comment in the TOC. The tense for the `Configure` title is inconsistent with the TOC entry.

**Review by craig-osterhout — CHANGES_REQUESTED** [review]

File: docker-hub/members.md (hunk 2 above)
Comment (jerae-duffin) [inline docker-hub/members.md:118]:
```suggestion
```

File: docker-hub/members.md [hunk 9]
```diff
@@ -115,6 +115,7 @@ To remove a member from a specific team:
 2. Click on the **Teams** tab and select the team from the list.
 3. Click the **X** next to the user’s name to remove them from the team.
 4. When prompted, click **Remove** to confirm.
+=======
```
Comment (jerae-duffin) [inline docker-hub/members.md:118]:
```suggestion
```

File: docker-hub/members.md (hunk 2 above)
Comment (jerae-duffin) [reply docker-hub/members.md]: Removed

File: docker-hub/members.md [hunk 10]
```diff
@@ -132,4 +132,5 @@ The CSV file contains the following fields:
 To export a CSV file of the organization's members:
 
 1. Navigate to **Organizations** in [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"}, and select your organization.
-2. In the **Members** tab, select **Export members** to download the CSV file.
\ No newline at end of file
+2. In the **Members** tab, select **Export members** to download the CSV file.
+>>>>>>> 5c68a7ad02a3a842aca984dcb72d13732506c06f
```
Comment (jerae-duffin) [inline docker-hub/members.md:136]:
```suggestion
```

File: docker-hub/members.md (hunk 3 above)
Comment (jerae-duffin) [reply docker-hub/members.md]: Removed

File: docker-hub/members.md (hunk 4 above)
Comment (jerae-duffin) [reply docker-hub/members.md:136]: Removed

File: docker-hub/repos/access/index.md (hunk 5 above)
Comment (jerae-duffin) [inline docker-hub/repos/access/index.md:5]:
```suggestion
```

File: docker-hub/repos/configure/index.md (hunk 6 above)
Comment (jerae-duffin) [reply docker-hub/repos/configure/index.md:5]: updated

Comment (jerae-duffin) [inline docker-hub/repos/configure/index.md:5]:
```suggestion
```

File: docker-hub/repos/index.md (hunk 7 above)
Comment (jerae-duffin) [reply docker-hub/repos/index.md:4]: Repositories or Repository? If we're referencing multiple repos?

Comment (jerae-duffin) [inline docker-hub/repos/index.md:4]:
```suggestion
title: Manage repositories
```

File: docker-hub/repos/configure/index.md (hunk 8 above)
Comment (jerae-duffin) [inline docker-hub/repos/configure/index.md:4]:
```suggestion
title: Configure repositories
```

File: docker-hub/repos/access/index.md [hunk 11]
```diff
@@ -0,0 +1,105 @@
+---
+description: Access repositories on Docker Hub
+keywords: Docker, docker, trusted, registry, accounts, plans, Dockerfile, Docker Hub, webhooks, docs, documentation, collaborators, viewing, searching, starring
+title: Access
```
Comment (jerae-duffin) [inline docker-hub/repos/access/index.md:4]:
```suggestion
title: Access repositories
```

**Review by craig-osterhout — APPROVED** [review]
LGTM

File: docker-hub/repos/index.md (hunk 7 above)
Comment (jerae-duffin) [reply docker-hub/repos/index.md]: Thanks Craig! Updated

**Review by ivan-californias — APPROVED** [review]
LGTM

File: docker-hub/repos/access/index.md [hunk 12]
```diff
@@ -0,0 +1,104 @@
...
+41459f052977: Pull complete
+fd44297e2ddb: Already exists
+centos:latest: The image you are pulling has been verified. Important: image verification is a tech preview feature and should not be relied on to provide security.
+Digest: sha256:d601d3b928eb2954653c59e65862aabb31edefa868bd5148a41fa45004c12288
+Status: Downloaded newer image for centos:latest
+```
+
+You now have an image from which you can run containers.
+
+## Starring Repositories
```
Comment (jerae-duffin) [inline docker-hub/repos/access/index.md:100]:
```suggestion
## Starring repositories
```

**Review by craig-osterhout — APPROVED** [review]
LGTM

**Pull request merged**